website/content/en/v0.12.0/AWS/user-data.md

```diff
@@ -0,0 +1,123 @@
+This document describes how you can customize the UserData that will be specified on your EC2 worker nodes, without using a launch template.
```

Comment:
This needs something like this

```
---
title: "User Data"
linkTitle: "User Data"
weight: 90
description: >
  Learn how Karpenter enables User Data
---
```

Reply:
Yeah, this was missing from previous website versions as well.  I can certainly add this.

Reply:
Looks like we did that on purpose:  https://github.com/aws/karpenter/commit/2873859fd8074e04969a60e347593dc83dc60172

Reply:
Removed the user-data.md page for all but the preview site.